Algorithm.Framework/QCAlgorithmFramework.cs

```diff
@@ -91,6 +98,19 @@ public override void PostInitialize()
                 AddUniverse(universe);
             }
 
+            if (DebugMode)
+            {
+                InsightsGenerated += (algorithm, data) => Log($"{Time}: {string.Join(" | ", data.Insights.OrderBy(i => i.Symbol.ToString()))}");
+            }
+
+            // emit warning message about using the framework with cash modelling
+            if (BrokerageModel.AccountType == AccountType.Cash)
+            {
+                Error("These models are currently unsuitable for Cash Modeled brokerages (e.g. GDAX) and may result in unexpected trades."
+                    + " To prevent possible user error we've restricted them to Margin trading. You can select margin account types with"
+                    + " SetBrokerage( ... AccountType.Margin) or substitute it for the ImmediateExecutionModel()");
```

Comment:
" or substitute it for the ImmediateExecutionModel()" -- this can be removed now; same issue for _all_ cash models :|